tests/array/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -711,9 +707,7 @@ $arr = array($var);
 $arr = [$var];
 $arr = [$var];
 $arr = [$foo => $bar];
-$arr = [
-    $foo => $bar
-];
+$arr = [$foo => $bar];
```

Comment:
Looks much better to me :+1:

Reply:
@czosel :+1: need rebase, i will do this in nexr future